src/components/markdown.js

```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
                 username={props.social.medium} /></>
             <><DisplaySocial base='https://www.youtube.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/youtube.svg'
                 username={props.social.youtube} /></>
+            <><DisplaySocial base='https://www.codechef.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/codechef.svg'
```

Comment:
Replace `base='https://www.codechef.com/c'` with `base='https://www.codechef.com/'` to make this work properly.